dartagnan/src/main/java/com/dat3m/dartagnan/parsers/program/visitors/boogie/StdProcedures.java

```diff
@@ -70,8 +69,11 @@ public static void handleStdFunction(VisitorBoogie visitor, Call_cmdContext ctx)
             // TODO: Implement this
             return;
         }
-        if (name.startsWith("memcpy") | name.startsWith("$memcpy")) {
-            // TODO: Implement this
+        if (name.equals("$memcpy.i8")) {
+            IExpr dst = (IExpr)ctx.call_params().exprs().expr(2).accept(visitor);
+            IExpr src = (IExpr)ctx.call_params().exprs().expr(3).accept(visitor);
+            IExpr length = (IExpr)ctx.call_params().exprs().expr(4).accept(visitor);
+            visitor.programBuilder.addChild(visitor.threadCount, EventFactory.Std.newMemCpy(dst, src, length));
```

Comment:
What are the parameters 0 and 1 that get skipped here? Can you add a comment saying what gets skipped?